translations/store-translations/integreat.json

```diff
@@ -177,6 +180,11 @@
       "defaultReleaseNote": "Uygulamayı sizin için daha da iyileştirdik ve bazı hataları giderdik. Uygulama ve yerel bilgiler hakkında geri bildirimde bulunarak Integreat'ın iyileştirilmesine katkıda bulunabilirsiniz. "
+    },
+    "uk": {
+      "name": "Integreat",
+      "description": "В додатку Інтегрейт ви можете знайти місцеву інформацію,події та консульскі центри. Ми допомогаємо Вам відслідковувати всю необхідну інформацію.Вся інформація була створена та оновлюється владою та організаціями.\n\nБЕЗКОШТОВНА ДОПОМОГА\nВи можете безкоштовно та без реклами користуватись додатком Інтегрейт. Цей додаток було створено благодійною організацією \"Від дверей до дверей\" та з допомогою іншіх міст, органів влади та соціальних спілок.\n\nПОСЛУГИ\nІнтегрейт є вашим цифровим гідом до найбліжчого міста і пропонує унікальні послуги:\n- Ви зможете знайти місцеву інформацію, події та інші послуги в одному додатку\n- Місцева інформация надходить з органів місцевої влади та постійно буде оновлюватись\n- Ви можете знайти потрібну інформацію з допомогою кнопки Пошук\n- У вкладці \"Послуги\" ви знайдете робочі вакансії та стужвання, які знаходяться біля вас\n- Додаток Інтегрейт показує вам цікаві події у вашому місті\n- Сповіщення на телефон дають Вам важливі новини стосовно вашого міста\n- Ви зможете поширювати інформацію вашим друзям\n- Якщо Ви залишите ваш відгук, це допоможе нам поліпшити додаток\n\nМИ НЕ ЗАПИСУЄМО ВАШІ ДАННІ\nДодаток Інтегрейт не записує ваші дані та як ви користуєтесь додатком. Ми тільки робимо запис як часто сторінка була відкрита на вибраній мові, але ми не записуємо і не передаємо ваші данні.\nІнтегрейт вимагає Ваш дозвіл на повне користування функцій додатка:\nЗберігання: Місцева інформація про ваше місто буде зберігатися на вашому телефоні. Ви зможете користуватись нею навіть без Інтернету\nЛокація:Під час користування картою та вибору Вашого міста в додатку, ви побачите вашу актуальну локацію.Додаток не аналізує вашу локацію.\nВай-Фай: Інтегрейт завантажує контент тільки коли ви підключені до Інтернету через вай фай. Для цього нам потрібно перевіряти чи Ви підключені до вай фаю",
+      "defaultReleaseNote": "Ми додали новинки та виправили деякі помилки. Допоможіть нам поліпшити Інтегрейт, залишивши свій відгук у додатку"
```

Comment:
Also dot missing in the end if not intended.

Reply:
Good catch :+1: